Modules/_datetimemodule.c

```diff
@@ -6933,8 +6983,7 @@ _datetime_exec(PyObject *module)
     if (capi == NULL) {
         goto error;
     }
-    PyObject *capsule = PyCapsule_New(capi, PyDateTime_CAPSULE_NAME,
-                                      datetime_destructor);
+    PyObject *capsule = PyCapsule_New(capi, PyDateTime_CAPSULE_NAME, NULL);
```

Comment:
Two `PyMem_Free(capi);` operations under this line can be removed from main and 3.13?